lightning/src/ln/features.rs

```diff
@@ -18,6 +18,35 @@
 //! the term "supports" is used in reference to a particular set of [`Features`]. That is, a node
 //! supports a feature if it advertises the feature (as either required or optional) to its peers.
 //! And the implementation can interpret a feature if the feature is known to it.
+//! 
+//! The following features are currently required in the LDK:
+//! - `VariableLengthOnion` - requires/supports variable-length routing onion payloads
```

Comment:
Thanks for listing required/optional features separately now! However, I'm not sure why for example `/supports` is still mentioned here?